src/components/Dialogs/RegionDialog/RegionDialogComponent.tsx

```diff
@@ -93,18 +95,25 @@ export class RegionDialogComponent extends React.Component<{ appStore: AppStore
             }
         }
 
+        let tooltips = region && region.regionId !== 0 && (
+            <React.Fragment>
+                <Tooltip content={`Region is ${region.locked ? "locked" : "unlocked"}`}>
+                    <AnchorButton intent={Intent.WARNING} minimal={true} icon={region.locked ? "lock" : "unlock"} onClick={region.toggleLock}/>
+                </Tooltip>
+                <Tooltip content={"Focus"}>
+                    <AnchorButton intent={Intent.WARNING} minimal={true} icon={"eye-open"} onClick={this.handleFocusClicked}/>
```

Comment:
this handler should also select the focused region. At the moment, it doesn't center on the correct region, but always on the selected region.

Reply:
When we select a region on the list, this region should be on the center, shouldn't it?

Reply:
sorry, I was looking at the wrong code. But it seems to be working now :thinking: @kswang1029 do you think clicking on the "center" button in the region list should _also_ select that region?

Reply:
Either way should be fine I think. We should consider the case of “active” region in widgets but so far I don’t see issues with it.

Reply:
Either way should be fine I think. We should consider the case of “active” region in widgets but so far I don’t see issues with it.